### DISCUSSION TITLE
Running RavenDB on Hetzner Block Stoarge - Any experience?

### DISCUSSION THREAD

**Discussion by tobiaszuercher** [ravendb/ravendb#17338] [Q&A]
hi
i was wondering inf anyone is running raven on hetzner block storage. a couple of years ago i was struggling with slow azure disks + raven.

anyone has some experience with it? fast enough? i'm considering it for an app which has quite a load, db is around 4 TB.

**Comment by ayende** [comment]
What sorts of IOPS / Bandwidth are they promising?
What is quite a load? How much RAM on the instance?
